cleanup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+read -p "Enter the region: " region
+export AWS_DEFAULT_REGION=$region
+
+# List of Terraform modules to apply in sequence
+targets=(
+  "module.eks_data_addons"
+  "module.eks_blueprints_addons"
+  "module.eks"
+  "module.vpc"
+)
+
+# Delete Karpenter resources
+kubectl delete --all nodeclaim
+kubectl delete --all nodepool
+kubectl delete --all ec2nodeclass
```

Comment:
Lets clean up all the Aerospike instances and their CRDs.

Reply:
aren't they removed when you uninstall the Helm release?